docs/content/launch.md

```diff
@@ -85,6 +85,22 @@ html_theme_options = {
 This will create a new Deepnote project every time you click the launch button.
 ```
 
+## Lightning Studio
+
+To add [Lightning Studio](https://lightning.ai) links to your page, add the following configuration:
```

Comment:
I would use https://lightning.ai/studios and call it Lightning Studios (plural) consistently throughout.